src/components/ContributionsEpicButtons.test.tsx

```diff
@@ -5,59 +5,48 @@
 import '@testing-library/jest-dom';
 import React from 'react';
 import { render, screen } from '@testing-library/react';
-import testData from './ContributionsEpic.testData';
+import { factories } from '../factories';
 
 import { ContributionsEpicButtons } from './ContributionsEpicButtons';
 
-describe('EpicButtons', () => {
+describe('ContributionsEpicButtons', () => {
     it('Renders the primary button', () => {
-        const variant = {
-            ...testData.content,
-            name: 'Example',
-            showTicker: false,
+        const variant = factories.variant.build({
             cta: {
                 text: 'Button text!',
-                baseUrl: 'https://support.theguardian.com/support',
             },
-        };
-        const tracking = testData.tracking;
+        });
+        const tracking = factories.tracking.build();
 
         render(
             <ContributionsEpicButtons
                 variant={variant}
                 tracking={tracking}
-                onOpenReminderClick={(): void => console.log('Reminder was clicked')}
+                onOpenReminderClick={jest.fn()}
             />,
         );
 
         expect(screen.getByText('Button text!')).toBeInTheDocument();
     });
-
     it('Renders nothing when no primary CTA', () => {
-        const variant = {
-            ...testData.content,
-            name: 'Example',
-            showTicker: false,
+        const variant = factories.variant.build({
             cta: undefined,
-        };
-        const tracking = testData.tracking;
+        });
+        const tracking = factories.tracking.build();
 
         const { queryByTestId } = render(
             <ContributionsEpicButtons
                 variant={variant}
                 tracking={tracking}
-                onOpenReminderClick={(): void => console.log('Reminder was clicked')}
+                onOpenReminderClick={jest.fn()}
             />,
         );
 
         expect(queryByTestId('epic-buttons')).toBeNull();
     });
 
     it('Does not render the reminder button when secondary CTA present', () => {
-        const variant = {
-            ...testData.content,
-            name: 'Example',
-            showTicker: false,
+        const variant = factories.variant.build({
             cta: {
```

Comment:
I think if there's no primary cta no other buttons will be rendered at all, so I'm trying to be explicit about what this test requires to pass (you're totally right, that removing this would have no effect on the test, but if we require it I like being explicit).